lib/plugin/allure.js

```diff
@@ -104,6 +104,9 @@ module.exports = (config) => {
     if (isHookSteps === false) {
       startMetaStep(step.metaStep);
       if (currentStep !== step) {
+        if (step.actor.includes('\u001b[36m')) {
```

Comment:
Could you write a line in comment about this tricky hook
We don't have allure tests so at least we should have a comment on this block
Why do you do this and why is that so

Reply:
Updated comment to reflect why we should do this